### ISSUE TITLE
Nested formula possible?

### ISSUE THREAD

**Issue by kuarig** [BernhardPosselt/foundryvtt-special-dice-roller#53]
Hello! First of all, let me tell you that I admire your work.

I was wondering if there was a way to add the dice to the formula by reference to a variable attribute (of an actor, in this case) or from a nested formula.

I've tried various things with my very limited knowledge, but it seems to only allow you to type the values ​​directly into the formula (from chat or macro syntax), numerically or by adding consecutive letters per value...

Thank you for all your effort, it would be very useful to me and my playing groups!

**Comment by BernhardPosselt** [comment]
Yes, no way to reference actors. You can however if you access the API https://github.com/BernhardPosselt/foundryvtt-special-dice-roller#rolling-dice-programmatically in a Macro for instance.

**Comment by kuarig** [comment]
Thank you for your immediate response Bernhard!

Man, I have no idea how to do that... Could you, please, give me a simple working sintax example? I could probably manage from there.

Thanks a lot!

**Comment by BernhardPosselt** [comment]
Something like this (PF2E system, everything after actor.system will depend on your system)
```js
await ChatMessage.create({
    content: game.specialDiceRoller.genesys.rollFormula(actor.system.details.level.value + 'a')
});
```
